anaconda.py

```diff
@@ -739,9 +423,11 @@ def cleanPStore():
     from pyanaconda import kickstart
     from pyanaconda import ntp
     from pyanaconda import keyboard
+    from pyanaconda import display
+    from pyanaconda import startup_utils
```

Comment:
Not sure if it matters here, but startup_utils are imported about 40 lines above.